src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class Lottery {
+    private static final int RANDOMIZERBOUND = 100;

```

Comment:
Please try to follow the naming convention for constants. Please see examples for constants names that consist from several words